tests/e2e/test_slack_post.py

```diff
@@ -4,7 +4,7 @@
 
 import certifi
 import pytest
-from slack import WebClient
+from slack.web.client import WebClient
```

Comment:
Not sure if I should change this to `from slack_utils import WebClient`

Open for thoughts.

Reply:
I'll leave it as is then. Thanks for the review!